components/colorslider/index.css

```diff
@@ -38,59 +60,140 @@
     z-index: 2;
   }
 
-  &.is-disabled {
-    pointer-events: none;
-  }
+	&.is-disabled {
+		pointer-events: none;
+
+		.spectrum-ColorSlider-gradient {
+			display: none;
+		}
+	}
 }
 
 .spectrum-ColorSlider--vertical {
   display: inline-block;
 
-  width: var(--spectrum-colorslider-vertical-width);
-  height: var(--spectrum-colorslider-vertical-default-length);
+	min-block-size: var(
+		--mod-color-slider-vertical-minimum-height,
+		var(
+			--mod-color-slider-minimum-length,
+			var(--spectrum-color-slider-minimum-length)
+		)
+	);
+	block-size: var(
+		--mod-color-slider-vertical-height,
+		var(--mod-color-slider-length, var(--spectrum-color-slider-length))
+	);
+	min-inline-size: 0;
+	inline-size: var(
+		--mod-color-slider-vertical-control-track-width,
+		var(
+			--mod-color-slider-control-track-height,
+			var(--spectrum-color-control-track-width)
+		)
+	);
 
-  .spectrum-ColorSlider-handle {
-    left: 50%;
-    top: 0;
-  }
+	.spectrum-ColorSlider-handle {
+		inset-inline-start: 50%;
+		inset-block-start: 0;
+	}
 }
 
 .spectrum-ColorSlider-handle {
-  left: 0;
-  top: 50%;
-
-  &:after {
-    border-radius: var(--spectrum-colorslider-handle-hitarea-border-radius);
-    width: var(--spectrum-colorslider-handle-hitarea-width);
-    height: var(--spectrum-colorslider-handle-hitarea-height);
-  }
+	inset-inline-start: 0;
+	inset-block-start: 50%;
```

Comment:
This has resulted in the colorHandles being on the right when viewed from RTL which is a change from what is was of the colorHandles being on the left regardless of text direction.

Reply:
Changing the orientation of the slider is the correct behavior for RTL.

Reply:
See https://github.com/adobe/spectrum-web-components/pull/3302

Reply:
@majornista Does this look good to you now?